3-enrich/emr-etl-runner/bin/snowplow-hadoop-fs-rmr-0.2.0.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Copyright (c) 2012-2018 Snowplow Analytics Ltd. All rights reserved.
+#
+# This program is licensed to you under the Apache License Version 2.0,
+# and you may not use this file except in compliance with the Apache License Version 2.0.
+# You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the Apache License Version 2.0 is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+
+# Author::    Joshua Beemster (mailto:support@snowplowanalytics.com)
+# Copyright:: Copyright (c) 2012-2018 Snowplow Analytics Ltd
+# License::   Apache License Version 2.0
+
+# Removes recursively a folder on HDFS
+hadoop fs -test -d $1
+if [ $? == 0 ]; then
+  hadoop fs -rm -r -skipTrash $1
+else
+  echo "Directory $1 does not exist"
```

Comment:
should we specify a 0 exit code in this case?

Reply:
It does it automatically so not sure its required - can add an exit 0 however.